File: 674 Longest Continuous Increasing Subsequence/longest_continuous_increasing_subsequence.py
class Solution:
    def findLengthOfLCIS(self, nums: List[int]) -> int:
        result = 0
        anchor = 0
        for i in range(len(nums)):
            if i > 0 and nums[i - 1] >= nums[i]:
                anchor = i
            result = result if result > i - anchor + 1 else i - anchor + 1
        return result

File: 0674 Longest Continuous Increasing Subsequence/LongestContinuousIncreasingSubsequence.java
class LongestContinuousIncreasingSubsequence {
    public int findLengthOfLCIS(int[] nums) {
        int max = 0;
        int anchor = 0;
        for (int i = 0; i < nums.length; i++) {
            if (i > 0 && nums[i - 1] >= nums[i]) {
                anchor = i;
            }
            max = Math.max(max, i - anchor + 1);
        }
        return max;
    }
}